client/widgets/CGarrisonInt.cpp

```diff
@@ -432,6 +432,7 @@ CGarrisonSlot::CGarrisonSlot(CGarrisonInt * Owner, int x, int y, SlotID IID, EGa
 
 	selectionImage = std::make_shared<CAnimImage>(graphics->getAnimation(imgName), 1);
 	selectionImage->disable();
+	selectionImage->center(creatureImage->pos.center());
```

Comment:
Here, as well as in windows with artifacts, selectionImage is used. But in all market windows, selection is implemented by drawing a rectangle. Which is adjusted in size to the image.
What do you think about making it the same everywhere?
I mean making a separate class in MiscWidgets that will draw a selection.

Reply:
I think this kind of widget will be suitable for the purposes here.